src/Websites/Websites/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Upcoming Release
 * Introduced secrets detection feature to safeguard sensitive data.
+* New parameter `UseZipDeploy` in cmdlet `Publish-AzWebApp` to deploy using the ZipDeploy API
+* `Publish-AzWebApp` now uses ZipDeploy to publish Function Apps
```

Comment:
Is this duplicated as the above one?

Reply:
The two are different, one documents the new UseZipDeploy parameter. The other documents that function apps will be deployed using ZipDeploy instead of OneDeploy.